Automattic/wp-calypso
# Add map url API request and update the map url to contatain the signiture on map change

#### Changes proposed in this Pull Request

* We want to server the map image request from using a signature in the url. 
* This PR update the editor so that we query the new map-url endpoint. 
And load the update the image with the url coming from the api respnse as expected. 

#### Testing instructions
Patch your .com sandbox with D46168-code
Related PR D46030-code

1. Load the previous calypso editor.

2. Visit the a create a new post url.  http://calypso.localhost:3000/post/example.com
3. Set the location. 
4. Notice that the map image url contains a signature query parameter. 

5. Add a title. Save the post.
6. Update the same location. 
7. Notice that it works as expected the new map appears as expected.
